### PR TITLE
[GH-1645] Update wfl.module.wgs/workflow-wdl to allow overriding read_length in CollectRawWgsMetrics.

### PR DESCRIPTION
**Pull request by tbl3rd**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-1644
- https://broadinstitute.atlassian.net/browse/GH-1645

### Changes

- Upgrade to `ExternalWholeGenomeReprocessing_v2.0.5`.
- Delete unused code from `wfl.util`.
- Clean up random cruft I noticed recently.

### Review Instructions

- The new WGS pipeline takes a while to run.

### CODE REVIEW

**Comment by tbl3rd** [comment]
```
tbl@wmf05-d86 ~/Broad/wfl # time make api TARGET=system
export CPCACHE=/Users/tbl/Broad/wfl/api/.cpcache;            \
	export WFL_WFL_URL=http://localhost:3000; \
	clojure  -M:parallel-test wfl.system.v1-endpoint-test | \
	tee /Users/tbl/Broad/wfl/derived/api/system.log
WARNING: Use of :paths external to the project has been deprecated, please remove: ../derived/api/src
WARNING: Use of :paths external to the project has been deprecated, please remove: ../derived/api/resources

Ran 33 tests containing 374 assertions.
0 failures, 0 errors.
api system finished on Fri Apr 1 13:11:52 EDT 2022

real	99m6.315s
user	0m38.981s
sys	0m6.641s
tbl@wmf05-d86 ~/Broad/wfl # 
```

**Comment by okotsopoulos** [comment]
> real	99m6.315s

Yowza.  From ~15 minutes to 1 hour 40 minutes?  Am I getting that right?

File: api/src/wfl/stage.clj [hunk 1]
```diff
@@ -22,10 +22,3 @@
   "A `stage` context object for logs."
   [stage]
   (:type stage))
-
-;; Goal: deprecate in favor of attaching `workloads/to-log` to logs.
-;;
-(defn log-prefix
```
Comment (okotsopoulos) [inline api/src/wfl/stage.clj:28]: Thank you!

**Review by okotsopoulos — APPROVED** [review]
I appreciate all the clean up.  I am worried about the 6.6x increase in system test runtime stemming from the WGS version bump though.

**Pull request merged**